get_pull_request.py

```diff
@@ -19,4 +19,4 @@ def get_pull_requests(owner, repo_name):
         print(f"Error fetching pull requests: {e}")
 
```

Comment:
20 : github.GithubException.UnknownObjectException if the provided owner or repo_name is invalid :  Use valid owner and repo names, and handle the exception with a more informative message or retry logic.  Potentially check if the repository is private and the token has the appropriate scopes.